docs/experiments/multiple-focus-metavariables.md

```diff
@@ -6,37 +6,47 @@ description: "With this rule, Semgrep matches all pieces of code captured by the
 
 # Using multiple focus metavariables
```

Comment:
Perhaps I'm nitpicking, but I would call this experiment "`focus-metavariable` with union semantics` or something like that.

Reply:
I think using the word "semantics" instead of "syntax" makes more sense here